### PR TITLE
Update installation.rst

### PR DESCRIPTION
**Pull request by randyh62**
fix broken link to https://rocm.docs.amd.com/projects/MIVisionX/en/latest/how-to/mivisionx-docker.html

### CODE REVIEW

**Review by kiritigowda — APPROVED** [review]

File: docs/install/installation.rst [hunk 1]
```diff
@@ -248,7 +248,7 @@ Docker
 
 MIVisionX provides developers with docker images for Ubuntu `20.04` / `22.04`. Using docker images developers can quickly prototype and build applications without having to be locked into a single system setup or lose valuable time figuring out the dependencies of the underlying software.
 
-Docker files to build MIVisionX containers and suggested workflow are `available <docker/README.md#mivisionx-docker>`_
+Docker files to build MIVisionX containers and suggested workflow are `available <../how-to/mivisionx-docker.html>`_
```
Comment (lpaoletti) [inline docs/install/installation.rst:251]: This sentence could use a period.

Comment (lpaoletti) [reply docs/install/installation.rst:251]: The 20.04` / `22.04`. in the previous paragraph shouldn't be in italics

Comment (lpaoletti) [reply docs/install/installation.rst:251]: Now rethinking the hyperlink that is being fixed. 
Ideally, the link name should be the same or similar to the destination. 
Perhaps this line could be:

For more information about the Docker files used to build MIVisionX containers and a suggested workflow, see **MIVisionX Docker documentation**. (hyperlink)

Comment (randyh62) [reply docs/install/installation.rst]: I have added a period and used Leo's suggested text, oh, and removed the italics

**Review by kiritigowda — APPROVED** [review]

**Pull request merged**